includes/autoloader.php

```diff
@@ -195,6 +195,10 @@ private static function normalize_class_name( $string, $delimiter = ' ' ) {
 
 	private static function init_classes_aliases() {
 		self::$classes_aliases = [
+			'Editor' => [
+				'replacement' => 'Core\Editor\Editor',
+				'version' => '2.6.0',
+			],
```

Comment:
Remove (I think it's from a wrong merge)

Reply:
Done